src/Pagination/Pagination.tsx

```diff
@@ -103,7 +102,7 @@ export const Pagination: FC<PaginationProps> = ({
         return (
           <PageBox
             active={activePage === i}
-            key={UUID()}
+            key={i}
```

Comment:
Indexing using the page number will work better in this case as the page number is unique and doesn't change between render

It also allow us to remove a dependency which isn't needed